### PR TITLE
[#10806] [UI issue][Help for instructor page][Questions section] non uniformity in the full stop at the end of the sentence

### LINKED ISSUE #10806: [UI issue][Help for instructor page][Questions section] non uniformity in the full stop at the end of the sentence
[UI issue][Help for instructor page][Questions section] non uniformity in the full stop at the end of the sentence

LIVE TEAMMATES V7.3.1

Steps to reproduce:
1. https://teammatesv4.appspot.com/web/front/help/instr
2. Go to the Questions section

Expected results:
Uniformity in all the answers in the page
The other answers have full stop at the end of the sentence

Actual results:
The full stops are missing in the answers under questions section
Screenshot 2020-10-23 at 16 36 31
Screenshot 2020-10-23 at 16 36 26

### PR DESCRIPTION
**Pull request by AdithyaNarayan**
Fixes #10806 

**Outline of Solution**
- Added punctuation to the bullet points

### CODE REVIEW

File: src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html [hunk 1]
```diff
@@ -591,28 +591,28 @@ <h4 *ngIf="displaySubsection(0, 10)">Setting Up Questions</h4>
...
       <li>
-        Specify the feedback path that should be used to generate the appropriate feedback recipients
+        Specify the feedback path that should be used to generate the appropriate feedback recipients.
       </li>
     </ol>
     <tm-question-edit-form *ngIf="questionsToCollapsed[QuestionsSectionQuestions.RANK_OPTIONS]"
-            [formModel]="exampleRankOptionQuestionModel"
-            [formMode]="QuestionEditFormMode.EDIT"
-            [isDisplayOnly]="true"
+                           [formModel]="exampleRankOptionQuestionModel"
```
Comment (ChooJeremy) [inline src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html:613]: Why do we add additional spacing?

**Review by ChooJeremy — COMMENTED** [review]
Looks good.

Would you mind updating for the other questions in the other sections as well? While most of them do have full stops after the sentences in lists, some don't (for example, `How do I create a comment on a response?` doesn't.

**Comment by rrtheonlyone** [comment]
@AdithyaNarayan any updates on this?

**Comment by AdithyaNarayan** [comment]
I've updated it for all the questions on the Instructor Help Page!

**Review by ChooJeremy — APPROVED** [review]

**Review by madanalogy — APPROVED** [review]

**Pull request merged**